package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "create-fusion-plugin",
+  "version": "0.0.4",
+  "main": "index.js",
+  "license": "MIT",
+  "files": ["templates", "bin"],
+  "bin": {
+    "create-fusion-plugin": "bin/cli.js"
+  },
+  "dependencies": {
+    "chalk": "^2.4.1",
+    "fusion-scaffolder": "https://github.com/kevingrandon/fusion-scaffolder"
```

Comment:
I think this is in fusionjs now?

Reply:
This is just a fork until https://github.com/fusionjs/fusion-scaffolder/pull/30 lands and is published with updates. Happy to change this if we land/publish that first.